src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
 // write your code here
+const population = [...document.querySelectorAll('.population')];
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+
+let total = 0;
+
+const setting = population
```

Comment:
There is a missing semicolon at the end of this line. It's important to maintain consistency in your code style.